src/components/SimpleMap/SimpleMap.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComposableMap, Geographies, Geography, Marker } from 'react-simple-maps';
+
+const geoUrl = 'https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json';
```

Comment:
make me slightly nervous to depend on an external resource like this. Is it reasonable to copy the json into our repo, or is that a bad idea?

Reply:
Hmm yeah, let me see what I can do about that

Reply:
have we looked into pulling in geomap panel? maybe we can use geomap with one layer and controls disabled 🤔

Reply:
We haven't, but in my past experience its not a trivial thing to use a panel outside of a dashboard